MeNext/Playing/MainController.cs

```diff
@@ -251,7 +251,7 @@ public void RequestSkip()
             // TODO: songFinished with actual song
             var task = Task.Run(async () =>
              {
-                 return await api.SongFinished(this.EventSlug, this.UserKey, "dummy");
+                return await api.SongFinished(this.EventSlug, this.UserKey, this.musicService.PlayingSong.UniqueId);
```

Comment:
I'm not sure this is correct. We shouldn't be calling SongFinished if the song hasn't actually finished. This was just a temp kludge, no?

@ConnorFoody Shouldn't we have an api skip function?